.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @ssbarnea @jamesregis
```

Comment:
This file should not be part of the PR, PR #23 still in review.

Reply:
When he will rebase it will dissaper.